src/Illuminate/Support/Collection.php

```diff
@@ -45,7 +45,9 @@ class Collection implements ArrayAccess, Arrayable, Countable, IteratorAggregate
      */
     public function __construct($items = [])
     {
-        $this->items = $this->getArrayableItems($items);
+        $this->items = $this->getArrayableItems(
+            func_num_args() === 1 ? $items : func_get_args()
```

Comment:
How would this change handle multiple arrays passed as separate arguments? e.g. `collect(['foo'], ['bar'])` ?